XO/XO.py

```diff
@@ -0,0 +1,116 @@
+from typing import List, Union, Tuple
+import random
+
+N = 3
+EMPTY = 'E'
+PLAYERS_SIGNS = ['X', 'O']
+
+
+def get_names() -> List[str]:
+    basic_text = 'Insert the name of player {index}\n'
+    inputs = []
+    for i in range(2):
+        text = basic_text.format(index=i)
+        inputs.append(input(text))
+    return inputs
+
+
+def get_action(current_player: str, board: List[list]) -> Tuple[int, int]:
+    """
+    :description: Talks with the user and get an input which represents his move, verify that the move is valid.
```

Comment:
No need for `:description:` prefix.
Simply start with `Talks with...`
And, add an empty line between the description area and the `:param:/:return:` area.